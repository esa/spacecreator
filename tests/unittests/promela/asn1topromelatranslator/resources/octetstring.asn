MYMODULE DEFINITIONS ::= BEGIN

MyOctetString ::= OCTET STRING (SIZE (6))

END