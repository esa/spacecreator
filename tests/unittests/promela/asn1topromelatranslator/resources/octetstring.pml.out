inline OctetStringElement_generate_value(OctetStringElement_gv)
{
    int OctetStringElement_tmp;
    if
    ::  atomic {
        select (OctetStringElement_tmp : 0 .. 255);
    }
    fi;
    OctetStringElement_gv = OctetStringElement_tmp;
}
inline MyOctetString_generate_value(MyOctetString_gv)
{
    int i;
    for(i : 0 .. 5)
    {
        OctetStringElement_generate_value(MyOctetString_gv.data[i]);
    }
}
